Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM mhart/alpine-node:0.10.48
 MAINTAINER butlerx <butlerx@notthe.cloud>
+ENV NODE_ENV=production
 ARG DEP_VERSION=latest
-RUN apk add --update git build-base python postgresql-client
-RUN mkdir -p /usr/src/app
+RUN apk add --update git build-base python postgresql-client &&\
+   mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-ADD . /usr/src/app
+COPY . /usr/src/app
```

Comment:
If you copy, changes won't be detected ?

Reply:
no they will be